charts/warpstream-agent/values.yaml

```diff
@@ -55,13 +55,13 @@ extraEnv: []
 extraArgs: []
 # Add additional args settings to the pod.
 
-resources: {}
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+resources:
+  # We recommend 2 vCPUs per Agent minimum, and at least 4 GiB of dedicated ram per vCPU.
+  requests:
+    cpu: 2
+    memory: 8192Mi
+  limits:
+    cpu: 2
```

Comment:
this [article](https://loft.sh/blog/how-to-set-up-kubernetes-requests-and-limits/) recommends to omit CPU limits:

> Imposing a limit on a pod will throttle its CPU usage even when there are actually spare CPU cycles available. This can reduce performance.

and they recommend to specify a memory limit equal to the memory request to avoid unexpected pod eviction.

Reply:
more details her: https://home.robusta.dev/blog/stop-using-cpu-limits